### PR TITLE
Examples fix

### PR DESCRIPTION
**Pull request by ckaran**
I made a mistake in naming a module earlier, and didn't catch it because I also forgot to add the module to `main.rs`, so it wasn't compiled... :expressionless:

### CODE REVIEW

**Comment by RustyYato** [comment]
You can merge things into your branches without my approval. I'll check things that merge into master, or if you have any questions.

**Review by RustyYato — APPROVED** [review]
Looks fine, please delete the branches after merging them.

**Comment by ckaran** [comment]
OK, I will do so in the future, and reserve reviews for when I ask you to merge to `master`, or when I have questions.  Thanks!

**Pull request merged**
